api/src/main/java/com/codeforcommunity/dto/imports/TreeBenefitImport.java

```diff
@@ -0,0 +1,247 @@
+    this.aqPm10Dep = aqPm10Dep;
+    this.aqSoxAvoided = aqSoxAvoided;
+    this.aqSoxDep = aqSoxDep;
+    this.aqVocAvoided = aqVocAvoided;
+    this.co2Avoided = co2Avoided;
+    this.co2Sequestered = co2Sequestered;
+    this.co2Storage = co2Storage;
+    this.electricity = electricity;
+    this.hydroInterception = hydroInterception;
+    this.naturalGas = naturalGas;
+  }
+
+  private TreeBenefitImport() {}
+
+  public String getSpeciesCode() {
+    return speciesCode;
+  }
+
+  public void setSpeciesCode(String speciesCode) {
+    this.speciesCode = speciesCode;
+  }
+
+  public Double getDiameter() {
+    return diameter;
+  }
+
+  public void setDiameter(Double diameter) {
+    this.diameter = diameter;
+  }
+
+  public Double getAqNoxAvoided() {
+    return aqNoxAvoided;
+  }
+
+  public void setAqNoxAvoided(Double aqNoxAvoided) {
+    this.aqNoxAvoided = aqNoxAvoided;
+  }
+
+  public Double getAqNoxDep() {
+    return aqNoxDep;
+  }
+
+  public void setAqNoxDep(Double aqNoxDep) {
+    this.aqNoxDep = aqNoxDep;
+  }
+
+  public Double getAqOzoneDep() {
+    return aqOzoneDep;
+  }
+
+  public void setAqOzoneDep(Double aqOzoneDep) {
+    this.aqOzoneDep = aqOzoneDep;
+  }
+
+  public Double getAqPm10Avoided() {
+    return aqPm10Avoided;
+  }
+
+  public void setAqPm10Avoided(Double aqPm10Avoided) {
+    this.aqPm10Avoided = aqPm10Avoided;
+  }
+
+  public Double getAqPm10Dep() {
+    return aqPm10Dep;
+  }
+
+  public void setAqPm10Dep(Double aqPm10Dep) {
+    this.aqPm10Dep = aqPm10Dep;
+  }
+
+  public Double getAqSoxAvoided() {
+    return aqSoxAvoided;
+  }
+
+  public void setAqSoxAvoided(Double aqSoxAvoided) {
+    this.aqSoxAvoided = aqSoxAvoided;
+  }
+
+  public Double getAqSoxDep() {
+    return aqSoxDep;
+  }
+
+  public void setAqSoxDep(Double aqSoxDep) {
+    this.aqSoxDep = aqSoxDep;
+  }
+
+  public Double getAqVocAvoided() {
+    return aqVocAvoided;
+  }
+
+  public void setAqVocAvoided(Double aqVocAvoided) {
+    this.aqVocAvoided = aqVocAvoided;
+  }
+
+  public Double getCo2Avoided() {
+    return co2Avoided;
+  }
+
+  public void setCo2Avoided(Double co2Avoided) {
+    this.co2Avoided = co2Avoided;
+  }
+
+  public Double getCo2Sequestered() {
+    return co2Sequestered;
+  }
+
+  public void setCo2Sequestered(Double co2Sequestered) {
+    this.co2Sequestered = co2Sequestered;
+  }
+
+  public Double getCo2Storage() {
+    return co2Storage;
+  }
+
+  public void setCo2Storage(Double co2Storage) {
+    this.co2Storage = co2Storage;
+  }
+
+  public Double getElectricity() {
+    return electricity;
+  }
+
+  public void setElectricity(Double electricity) {
+    this.electricity = electricity;
+  }
+
+  public Double getHydroInterception() {
+    return hydroInterception;
+  }
+
+  public void setHydroInterception(Double hydroInterception) {
+    this.hydroInterception = hydroInterception;
+  }
+
+  public Double getNaturalGas() {
+    return naturalGas;
+  }
+
+  public void setNaturalGas(Double naturalGas) {
```

Comment:
I really hope you didn't have to write all of these manually

Reply:
[Fortunately not](https://www.jetbrains.com/help/idea/generating-code.html#generate-getters-setters) but others required some copy-pasting :(